addons/account_payment/wizards/payment_refund_wizard.py

```diff
@@ -29,6 +29,8 @@ class PaymentRefundWizard(models.TransientModel):
     support_refund = fields.Selection(
         string="Type of Refund Supported",
         selection=[('full_only', "Full Only"), ('partial', "Partial")],
+        # Bypassing access rights checks for the payment provider model for all users  
```

Comment:
```suggestion
        # Bypassing access rights checks for the payment provider model for all users
```
Try configuring your editor in a that trailing whitespace gets highlighted, or use a linter like [Ruff](https://docs.astral.sh/ruff/rules/trailing-whitespace/).